db/schema.rb

```diff
@@ -407,6 +407,12 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "hyrax_flexible_schemas", force: :cascade do |t|
```

Comment:
I'm not sure why this came in... there must have been a prior migration that hadn't been run and saved?

Reply:
It needs to be manually removed or it will break future migrations on new apps. Just deleting these lines will do the trick.